cime/config/e3sm/machines/config_machines.xml

```diff
@@ -1922,21 +1922,24 @@
         <command name="load">cmake/3.11.4</command>
       </modules>
       <modules compiler="intel">
-        <command name="load">intel/18.0.0</command>
+        <command name="load">intel/19.0.5</command>
       </modules>
       <modules compiler="pgi">
         <command name="load">pgi/18.10</command>
       </modules>
       <modules mpilib="mvapich2">
         <command name="load">mvapich2/2.3.1</command>
       </modules>
-      <modules mpilib="impi">
-        <command name="load">intelmpi/2018</command>
+      <modules mpilib="impi" compiler="intel">
+	<command name="load">intelmpi/2019u5</command>
```

Comment:
This upgrade might cause issues with MPI I/O like noted in the discussion for this PR

Reply:
Yes, I can ask the support to compile an older version of Intel MPI with this new compiler if we know which version introduced this bug.